metafix/src/main/java/org/metafacture/metafix/FixPath.java

```diff
@@ -0,0 +1,381 @@
+/*
+ * Copyright 2022 Fabian Steeg, hbz NRW
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.metafacture.metafix;
+
+import org.metafacture.metafix.Value.Array;
+import org.metafacture.metafix.Value.Hash;
+import org.metafacture.metafix.Value.ReservedField;
+import org.metafacture.metafix.Value.TypeMatcher;
```

Comment:
Hm, I'm undecided if I like these `import`s of nested types; in particular as they refer to a sibling class. Anyway, it's easier for now not to change all references.